include/SFML/Window/Keyboard.hpp

```diff
@@ -153,6 +158,158 @@ class SFML_WINDOW_API Keyboard
+        ScanS,                 ///< Keyboard s and S key
+        ScanT,                 ///< Keyboard t and T key
+        ScanU,                 ///< Keyboard u and U key
+        ScanV,                 ///< Keyboard v and V key
+        ScanW,                 ///< Keyboard w and W key
+        ScanX,                 ///< Keyboard x and X key
+        ScanY,                 ///< Keyboard y and Y key
+        ScanZ,                 ///< Keyboard z and Z key
+        ScanNum1,              ///< Keyboard 1 and ! key
+        ScanNum2,              ///< Keyboard 2 and @ key
+        ScanNum3,              ///< Keyboard 3 and # key
+        ScanNum4,              ///< Keyboard 4 and $ key
+        ScanNum5,              ///< Keyboard 5 and % key
+        ScanNum6,              ///< Keyboard 6 and ^ key
+        ScanNum7,              ///< Keyboard 7 and & key
+        ScanNum8,              ///< Keyboard 8 and * key
+        ScanNum9,              ///< Keyboard 9 and ) key
+        ScanNum0,              ///< Keyboard 0 and ) key
+        ScanEnter,             ///< Keyboard Return (ENTER) key
+        ScanEscape,            ///< Keyboard Escape key
+        ScanBackspace,         ///< Keyboard Backspace key
+        // TODO above it's BackSpace, but is it correct? What do we use here?
+        ScanTab,               ///< Keyboard Tab key
+        ScanSpace,             ///< Keyboard Space key
+        ScanHyphen,            ///< Keyboard - and _ key
+        ScanEquals,            ///< Keyboard = and +
+        ScanLBracket,          ///< Keyboard [ and { key
+        ScanRBracket,          ///< Keyboard ] and } key
+        ScanBackslash,         ///< Keyboard \ and | key
+        // TODO capitalisation
+        ScanDash,              ///< Keyboard Non-US # and ~
+        ScanSemicolon,         ///< Keyboard ; and : key
+        // TODO capitalisation
+        ScanQuote,             ///< Keyboard ' and " key
+        ScanGraveAccent,       ///< Keyboard ` and ~ key
+        ScanComma,             ///< Keyboard , and < key
+        ScanPeriod,            ///< Keyboard . and > key
+        ScanForwardSlash,      ///< Keyboard / and ? key
+        ScanF1,                ///< Keyboard F1 key
+        ScanF2,                ///< Keyboard F2 key
+        ScanF3,                ///< Keyboard F3 key
+        ScanF4,                ///< Keyboard F4 key
+        ScanF5,                ///< Keyboard F5 key
+        ScanF6,                ///< Keyboard F6 key
+        ScanF7,                ///< Keyboard F7 key
+        ScanF8,                ///< Keyboard F8 key
+        ScanF9,                ///< Keyboard F9 key
+        ScanF10,               ///< Keyboard F10 key
+        ScanF11,               ///< Keyboard F11 key
+        ScanF12,               ///< Keyboard F12 key
+        ScanF13,               ///< Keyboard F13 key
+        ScanF14,               ///< Keyboard F14 key
+        ScanF15,               ///< Keyboard F15 key
```

Comment:
Apparently, scan codes go up to F25. It's very unlikely that someone will use them though, should we bother?

Reply:
I would, support all the possible keys, so we don't have to readd them later.

Reply:
@eliasdaler Do you remember where you read that?

Reply:
I'll actually back paddle on that one. I'd rather add keys when we actually have a keyboard to tests such keys with, than blindly adding something, that nobody can confirm.